plugins/modules/slack.py

```diff
@@ -32,18 +32,21 @@
   domain:
     type: str
     description:
-      - Slack (sub)domain for your environment without protocol. (For example V(example.slack.com).) In Ansible 1.8 and beyond,
-        this is deprecated and may be ignored. See token documentation for information.
+      - When using new format 'Webhook token' and WebAPI tokens: this can be V(slack.com) or V(slack-gov.com) and is ignored otherwise.
```

Comment:
Same here:
```suggestion
      - "When using new format 'Webhook token' and WebAPI tokens: this can be V(slack.com) or V(slack-gov.com) and is ignored otherwise."
```